Packs/MicrosoftGraphCalendar/Integrations/MicrosoftGraphCalendar/README.md

```diff
@@ -37,6 +37,11 @@
 </li>
 <li>Click <strong>Test</strong> to validate the new instance.</li>
 </ol>
+
+<h2>Use a self deployed Azure application</h2>
+<p>To use a self configured Azure application, a new Azure App Registration should be added in the Azure Portal. To add such registration, refer to the following Microsoft article:
```

Comment:
please also mention how to fill in the integration params, like what should be filled in id, key and token